<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>826</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1262</width>
    <height>826</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Config Trail Camera</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupCalcType_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>70</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>Connect to board</string>
    </property>
    <widget class="QLabel" name="label_15">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose port</string>
     </property>
    </widget>
    <widget class="QComboBox" name="portComboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="connect_butt">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>connect</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/play.png</normaloff>icons/play.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>refresh</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lteRstButt">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>LTE reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="camRstButt">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TrailCamera reset</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuration of TrailCamera</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wake up time</string>
     </property>
    </widget>
    <widget class="QComboBox" name="wakeupcomboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Send Photo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Photo resolution</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sendPhotoComboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="resolutioncomboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Photo quality</string>
     </property>
    </widget>
    <widget class="QComboBox" name="qualitycomboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>491</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>Take photo</string>
    </property>
    <widget class="QPushButton" name="takephotopushButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Take a photo</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="photoView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>471</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>SMS Configuration</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Phone number</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addPhoneNum">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add phone number</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="batterycheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Battery level</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chargingcheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Charging status</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="batterykifecheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Remaining battery time</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendSMSpushButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QComboBox" name="phoneNumberComboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="sendAutoSMScheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>251</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Send SMS when motion captured</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Network Configuration</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>APN</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="apnLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="urlLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>URL</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Shared Key</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sharedKeyLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="saveConfButton">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>700</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save configuration</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>100</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Get info status</string>
    </property>
    <widget class="QPushButton" name="getbatterypushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Get battery level</string>
     </property>
    </widget>
    <widget class="QLabel" name="batterylabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>battery</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gettimepushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Get remaning time</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getchargingpushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Get charging status</string>
     </property>
    </widget>
    <widget class="QLabel" name="remaininglabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>remaining time</string>
     </property>
    </widget>
    <widget class="QLabel" name="charginglabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>chargingstatus</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getSingalLevelButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Get signal level</string>
     </property>
    </widget>
    <widget class="QLabel" name="signalLevellabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>signal level</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="defaultConfButt">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>730</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Default configuration</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1182</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="iconText">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Terminate the program and exit.</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Options...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
